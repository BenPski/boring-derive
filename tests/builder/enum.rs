use boring_derive::Builder;

#[derive(Builder)]
enum Example {
    Item(usize),
}

fn main() {}
