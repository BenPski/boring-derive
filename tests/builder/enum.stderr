error: deriving builder pattern not supported for enums
 --> tests/builder/enum.rs:4:6
  |
4 | enum Example {
  |      ^^^^^^^
